apps/sms/lib/bridge/bridge.js

```diff
@@ -32,13 +32,27 @@ module.exports = Client;
 /**
```

Comment:
> So is it possible to replace the bridge with client in startup and replace the bridge with service/client ?

You mean lazy load both client and service right in the startup instead of single bridge or ...? If so, I'm a bit hesitating to do this in the patch that can be uplifted to v2.5, since it will likely affect ```fullyLoaded``` a bit more (size of ```bridge.min.js``` is ```15.6KB```, size of both ```client.min.js``` (```12.8KB```) and ```service.min.js```(```12KB```) will be ```24.8KB``` and they contain a bunch of duplicating stuff that should be parsed as well).

What about doing this in the patch that will get rid of ```ActivityShim``` from window (if we land it in v2.6 eventually), @steveck-chung ?